assets/css/mob-about.css

```diff
@@ -0,0 +1,114 @@
+@media screen and (max-width: 768px) {
+  @import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    font-family: "Lato", sans-serif;
+  }
+
+  .mob-header {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    margin-left: 23px;
+  }
+
+  .first-text {
+    width: 100vw;
+    text-align: center;
+  }
+
+  .page-intro {
+    text-align: center;
+    line-height: 35px;
+  }
+
+  .second-text {
+    width: 80vw;
+  }
+
+  .logo-text {
+    width: 80%;
+  }
+
+  .summit-text {
+    text-align: center;
+  }
+
+  .summit-pics-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    gap: 12px;
+  }
+
+  .partner-page {
+    height: 40vh;
+  }
+
+  .partner-content {
+    align-items: center;
+    padding: 10% 2% 2%;
+  }
+
+  .partner-list {
+    width: 100%;
+    padding-left: 5%;
+  }
+
+  .partner-pic {
+    width: 85%;
+  }
```

Comment:
- [ ] Please ensure that your partner logo image do not overlap, please ensure you follow the design guidelines, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![student about partners texts](https://github.com/Kwamay/Capstone-Project/assets/43528594/be493e06-7c3c-4f4e-a35d-935abce2b13d)


</td>
      <td>

![Design about image](https://github.com/Kwamay/Capstone-Project/assets/43528594/f2fb65e5-6ea3-45bd-8ade-d9c767117e6e)

</td>
    </tr>
  </table>